gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/store/ManagedDirectBufferAllocator.java

```diff
@@ -79,12 +64,6 @@ public static long defaultMaxMemory() {
         .getTotalPhysicalMemorySize() << 2) / 5 + 7) >>> 3) << 3;
   }
 
-  public ManagedDirectBufferAllocator initialize() {
-    this.logger = initLogger();
-    DirectBufferAllocator.setInstance(this);
-    return this;
-  }
```

Comment:
Oh ok, I thought the base would initialize the derived when loaded at runtime, but I guess this should be tagged as "@Override" and put back in place?